core/components/atoms/dropdown/__tests__/Dropdown.test.tsx

```diff
@@ -486,6 +487,29 @@ describe('Dropdown component with search', () => {
     fireEvent.click(dropdownTrigger);
     expect(screen.getByPlaceholderText('Custom search text')).toBeInTheDocument();
   });
+
+  it('when no record is available', () => {
+    const { getByTestId, queryByTestId } = render(<Dropdown options={undefined} withSearch={true} />);
```

Comment:
Please write proper description for the test case. Also please do not pass undefined to the options